samples/pipeline.yaml

```diff
@@ -58,6 +67,9 @@ resources:
     deletion_protection: true
 
 dag:
+  # Specify the Airflow version of the operators used by the DAG. Defaults to Airflow 1 when unspecified.
+  airflow_version: 1
```

Comment:
Prefer defaulting to Airflow 2

Reply:
Done! This `pipeline.yaml` sample file was written with Airflow 1 operators and used for all the unit tests.

I'll need to write another sample YAML file that uses Airflow 2 operators, and test that in addition to the old sample file. I've got it tracked via #137. I'll follow up with it very shortly! 😅

Reply:
That's for the follow up YAML file I'm writing (say `samples/pipeline.airflow2.yaml` ) that uses Airflow 2 operator names. It's because the names of some operators have changed in Airflow 2.

For example, the `BigQueryOperator` used in `samples/pipeline.yaml` exists in Airflow 1 but not in Airflow 2 (the equivalent is `BigQueryGetDataOperator`). If we set `airflow_version: 2` in samples/pipeline.yaml, the DAG generator will fail because among its operators is an Airflow 1 name that doesn't exist in Airflow 2.

Reply:
Issue is fixed now with a new `samples/pipeline.yaml` in #138. Resolving this